setup.cfg

```
@@ -14,16 +14,19 @@ zip_safe = False
 include_package_data = True
 install_requires =
     pydantic
-    ga4gh.vrsatile.pydantic >=0.0.11
+    ga4gh.vrsatile.pydantic ==0.0.11
     ga4gh.vrs[extras] >=0.7.2
     biocommons.seqrepo
-    gene-normalizer >=0.1.27
-    uta-tools >=0.1.0
+    gene-normalizer ==0.1.27
+    uta-tools >=0.1.2
 
 tests_require =
     pytest
     pytest-cov
     pytest-asyncio
 
+[options.package_data]
+fusor/examples = **
```

I think you'll want:
```suggestion
fusor =
    examples/*
```